roles/glance/templates/etc/glance/policy.json

```diff
@@ -7,7 +7,7 @@
     "get_image": "",
     "get_images": "",
     "modify_image": "",
-    "publicize_image": "role:admin",
+    "publicize_image": ["role:admin", "role:cloud_admin"],
```

Comment:
Ah, I went poking around to find docs on the format. I failed. Can you link to those?